app/routes/circulars/circulars.lib.ts

```diff
@@ -76,6 +76,25 @@ export function formatDateISO(date: number) {
   return new Date(date).toISOString().replace(/\.\d+Z$/, 'Z')
 }
 
+/** convert a ms value to mm-dd-yyy format */
+export function formatDateMMDDYYYY(date: number) {
+  const dateObj = new Date(date)
+  var dateStr = dateObj.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+  })
+  return dateStr.replace(/\//g, '-') // replace / with -
+}
+
+/** convert a date in format mm-dd-yyyy to ms since 01/01/1970 */
+export function formatDateEpoch(date: string | number) {
+  if (typeof date === 'string') {
+    return new Date(date).getTime()
+  }
+  return date
```

Comment:
I think it's a bit too fancy for it to take either a UNIX timestamp or a string. Let's just have this take a string, so that this function is simply the inverse of `formatDateMMDDYYYY `.

```suggestion
export function formatDateEpoch(date: string) {
  return new Date(date).getTime()
```

Reply:
the rationale I had behind this was so the search function was more flexible when defining the startTime and endTime constants so it could intake either the unix format as a number or the human-readable/iso8601 format as a string (see [here](https://github.com/tylerbarna/gcn.nasa.gov/blob/6d1838715fdd7b4da9b28ef040fb4e1ddfc856de/app/routes/circulars/circulars.server.ts#L70)), though I see why it's better to only keep it as one type. Do you think it would be better to keep it as the string up to the point of defining it in the search function?